packages/sui-js/README.md

```diff
@@ -96,6 +96,11 @@ import {parseQueryString} from '@s-ui/js/lib/string'
 console.log(parseQueryString('?targetPage=pta')) // {targetPage: "pta"}
 
 
+import {arrayToCommaQueryString} from '@s-ui/js/lib/string'
+
+console.log(arrayToCommaQueryString({userId: 1, adId: 2, products: [3, 4, 5]})) // 'a=1&b=2&c=3,4,5'
```

Comment:
I would expect `userId=1&adId=2&products=3,4,5` to be the output.

Reply:
Copy and paste fail... 🤓

Reply:
FIXED!